specs/windows/disk_info.table

```diff
@@ -0,0 +1,16 @@
+table_name("disk_info")
+description("Retrieve basic information about the physical disks of a system.")
+schema([
+    Column("partitions", INTEGER, "Number of detected partitions on disk."),
+    Column("disk_index", INTEGER, "Physical drive number of the disk."),
+    Column("type", TEXT, "The interface type of the disk."),
+    Column("device_id", TEXT, "The unique identifier of the drive on the system."),
```

Comment:
Similarly here, perhaps just `id` would be sufficient?

Reply:
I think that should be ok! Let me try changing it and making sure it doesn't complain like it did with i had "index" in there